test/k8sT/istio.go

```diff
@@ -90,13 +93,22 @@ var _ = Describe("K8sIstioTest", func() {
 		res.ExpectSuccess("unable to create namespace %q", istioSystemNamespace)
 
 		By("Creating the Istio resources")
+
+		res = kubectl.Apply(istioCRDYAMLPath)
+		res.ExpectSuccess("unable to create Istio CRDs")
+
+		By("Waiting for Istio CRDs to be ready")
+		err := kubectl.WaitForCRDCount("istio.io|certmanager.k8s.io", 53, helpers.HelperTimeout)
```

Comment:
53 looks really magical here, can we move it out to a variable with descriptive name?

Reply:
Its the number of Istio CRDs created by version 1.1.2. I'll add a variable if any other updates are needed, thanks!